modules/thermal_hydraulics/doc/content/modules/thermal_hydraulics/modeling_guide/recuperated_brayton_cycle/recuperated_brayton_cycle.md

```diff
@@ -0,0 +1,222 @@
+# Recuperated Brayton Cycle
+
+## Introduction
+
+As shown in the [Brayton Cycle modeling guide](modules/thermal_hydraulics/modeling_guide/brayton_cycle/brayton_cycle.md), and the given input files [closed_brayton_cycle.i](thermal_hydraulics/test/tests/problems/brayton_cycle/closed_brayton_cycle.i) and [open_brayton_cycle.i](thermal_hydraulics/test/tests/problems/brayton_cycle/open_brayton_cycle.i), a Brayton Cycle Power Conversion Unit (PCU) consists of a motor, compressor, turbine, and generator all coupled by a single shaft.
```

Comment:
@licharlot what's your opinion on line-wrapping? Breaking is my preference, but I don't feel *too* strongly about it.

Reply:
The output will be unaffected by the line breaking. A new paragraph will only result if you put a blank line between lines.